github-workflows/.github/workflows/python-package.yml

```diff
@@ -38,9 +38,9 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install flake8 isort
+        python -m pip install ruff==0.7.3
```

Comment:
Why is a different version hardcoded here and a different one in the requirements?

Reply:
Incorrect one in requirements, updated 👍